forch/forchestrator.py

```diff
@@ -194,17 +194,17 @@ def initialized(self):
         """If forch is initialized or not"""
         return self._initialized
 
-    def _process_device_placement(self, eth_src, device_placement):
+    def _process_device_placement(self, eth_src, device_placement, persisted=False):
```

Comment:
use default  (also, should be consistent -- either used named arg or positional)

Reply:
This is function definition?